quickstarts/microsoft.mobilenetwork/mobilenetwork-create-new-slice/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "value": "eastus"
+        },
+        "existingMobileNetworkName": {
+            "value": "GET-PREREQ-existingMobileNetworkName"
+        },
+        "sliceName": {
+            "value": "GEN-UNIQUE"
+        },
+        "sst": {
+            "value": 1
+        },
+        "sd": {
```

Comment:
remove this so the defaultValue can be tested

Reply:
Done